static/app/views/metrics/metricQueryContextMenu.tsx

```diff
@@ -162,8 +163,8 @@ export function MetricQueryContextMenu({
               router
             );
           } else {
-            // TODO(telemetry-experience): As soon as the span-based-metrics data has an unique identifier, we should use it here
-            navigateTo(`/settings/projects/:projectId/metrics/`, router);
+            const {name} = parseMRI(metricsQuery.mri) ?? {};
+            navigateTo(`/settings/projects/:projectId/metrics/${name}/edit/`, router);
```

Comment:
This logic branch will now be executed for all non-custom metrics. Also those witch are in namespaces, that are not covered by span extraction rules e.g. `transaction.duration`. Is that intentional?

Reply:
I thought we would still update the field on the metrics page so that the list is aligned with the table we display in settings. But this is not the intention, is there any function I can use to filter this?